### PR TITLE
Invoking Sevtug is now a global hallucination effect instead of mind control

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Invoking Sevtug now causes massive hallucinations, brain damage, confusion, and dizziness for all non-servant humans on the same zlevel as the invoker.
rscdel: Invoking Sevtug is no longer mind control. RIP mind control 2016-2016.
/:cl:

This will seriously fuck people up, and the closer they are to the invoker the worse the effects are.
It also includes a sound and color flash for all affected.

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -1041,34 +1041,55 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
...
 	invocations = list("V pnyy hcba lbh, Sevtug!!", "Yrg lbhe cbjre fung-gre gur fnavgl bs gur jrnx-zvaqrq!!", "Yrg lbhe graqevyf ubyq fjnl bire nyy!!")
 	channel_time = 150
 	required_components = list("belligerent_eye" = 3, "vanguard_cogwheel" = 3, "guvax_capacitor" = 6, "hierophant_ansible" = 3)
 	consumed_components = list("belligerent_eye" = 3, "vanguard_cogwheel" = 3, "guvax_capacitor" = 6, "hierophant_ansible" = 3)
-	usage_tip = "Completely silent and functions on silicon lifeforms. There will be no indication to those near the controlled."
+	usage_tip = "Causes brain damage, hallucinations, confusion, and dizziness in massive amounts."
 	tier = SCRIPTURE_REVENANT
+	var/list/mindbreaksayings = list("\"Bu, terng. V trg gb funggre fbzr zvaqf.\"", "\"Zber zvaqf gb pehfu.\"", \
+	"\"Ernyyl, guvf vf nyzbfg obevat.\"", "\"Abar bs gurfr zvaqf unir nalguvat vagrerfgvat va gurz.\"", "\"Znlor V pna vafgvyy n yvggyr ovg bs greebe va guvf bar.\"", \
+	"\"Jung n jnfgr bs zl cbjre.\"", "\"V'z fher V pbhyq whfg pbageby gurfr zvaqf vafgrnq, ohg gurl arire nfx.\"")
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Don't think I didn't miss the last saying here! _narrows eyes_

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Sevtug is a very snarky asshole

**Comment by Kromgar** [comment]
Thank god that mind control was super op

**Comment by ChangelingRain** [comment]
Don't think this isn't just as OP, it's just not directed at someone.

**Comment by AdamElTablawy** [comment]
this is honestly a little worse but it also fucks over your fellow cultists

**Comment by ChangelingRain** [comment]
No it doesn't, it won't affect servants.

File: code/game/gamemodes/clock_cult/clock_scripture.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: remove ohg

**Comment by KorPhaeron** [comment]
This might actullally be more op than the other one guess we will see

**Pull request merged**